doc/source/whatsnew/v1.4.0.rst

```diff
@@ -142,7 +143,8 @@ These are bug fixes that might have notable behavior changes.
 Inconsistent date string parsing
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``dayfirst`` option of :func:`to_datetime` isn't strict, and this can lead to surprising behaviour:
+The ``dayfirst`` option of :func:`to_datetime` isn't strict, and this can lead to sur
+ising behaviour:
```

Comment:
Looks like an accidental change?

Reply:
It was, sorry about that. I've reverted it in a new commit; feel free to squash it when merging.